test/kafka-ingest-open-loop/mzcompose.py

```diff
@@ -109,6 +114,12 @@ def workflow_kafka_ingest_open_loop(
             "--disable-persistent-system-tables-test",
         ]
 
+    if args.s3_storage is not None:
+        options.extend([
+            "--persist-storage-enabled",
+            f"--persist-storage=s3://mtlz-test-persist-1d-lifecycle-delete/{args.s3_storage}",
```

Comment:
I orignally had this flag named `--persist-storage` and you had to specify the full path, but I felt it best to optimize for our internal testing which will always be running from bin/scratch (so no arn necessary) and pointed at this bucket. if that seems objectionable, I feel like adding `--persist-storage` back into this PR as an additional option might be a good compromise

Reply:
Seems right to me. I can't imagine we're going to have hordes of external contributors beating down our door because they can't run this test.